db/fixtures/miq_product_features.yml

```diff
@@ -6088,6 +6092,10 @@
         :description: Refresh Block Storage Managers
         :feature_type: control
         :identifier: ems_block_storage_refresh
+      - :name: Re-check Authentication Status
+        :description: Re-check Authentication Status of storage manager
```

Comment:
`Block Storage Manager` to follow the convention here.

Reply:
In ui-classic, it is in block storage toolbar, but anyways, i am in process of combining block and object storage to one for another enhancement, then @agrare going to refactor this file.